homeassistant/const.py

```diff
@@ -266,6 +266,7 @@ class Platform(StrEnum):
 LEGACY_CONF_WHITELIST_EXTERNAL_DIRS: Final = "whitelist_external_dirs"
 CONF_XY: Final = "xy"
 CONF_ZONE: Final = "zone"
+DEFAULT_VERIFY_SSL: Final = True
```

Comment:
We don't keep default in our core constants. There is 1 exception, which is the unnamed device.

I'm not sure if this is a pattern we should aim for.